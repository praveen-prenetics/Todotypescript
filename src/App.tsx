import React from 'react';
//import logo from './logo.svg';
import './App.css';
import './style.css';
import { BrowserRouter, Route, Routes } from "react-router-dom";
import Login from './Login';
import Todo from './Todo';
import Register from './Register';

function App() {
  return (
    <BrowserRouter>
      <Routes>

        <Route path="/" element={<Login />} />
        <Route path="/register" element={<Register />} />
        <Route path="/Todo" element={<Todo />} />

      </Routes>

    </BrowserRouter>
  );
}

export default App;
