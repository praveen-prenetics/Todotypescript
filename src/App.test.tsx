import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';


// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });



describe('snapshot testing for App page', () => {
  it('should render', () => {
    const view = render(<App />);
    expect(view).toMatchSnapshot();
  });
});