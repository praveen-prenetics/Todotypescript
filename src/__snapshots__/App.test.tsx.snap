// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing for App page should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <form>
          <div
            class="top"
          >
            <div>
              <p
                class="proname"
              >
                Todos
              </p>
            </div>
          </div>
          <div
            class="container"
          >
            <div
              class="content"
            >
              <div
                class="topic"
              >
                <p
                  id="title"
                >
                  <strong>
                    Sign in
                  </strong>
                </p>
                <div
                  id="main"
                >
                  <input
                    class="uname"
                    id="username"
                    placeholder="Username"
                    type="email"
                    value=""
                  />
                  <input
                    class="pwd"
                    id="password"
                    placeholder="Password"
                    type="password"
                    value=""
                  />
                  <div>
                    <button
                      id="button"
                    >
                      <b>
                        Sign in
                      </b>
                    </button>
                  </div>
                  <div
                    class="direct"
                  >
                    <a
                      class=""
                      href="/register"
                    >
                      New here? Register now
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <form>
        <div
          class="top"
        >
          <div>
            <p
              class="proname"
            >
              Todos
            </p>
          </div>
        </div>
        <div
          class="container"
        >
          <div
            class="content"
          >
            <div
              class="topic"
            >
              <p
                id="title"
              >
                <strong>
                  Sign in
                </strong>
              </p>
              <div
                id="main"
              >
                <input
                  class="uname"
                  id="username"
                  placeholder="Username"
                  type="email"
                  value=""
                />
                <input
                  class="pwd"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <div>
                  <button
                    id="button"
                  >
                    <b>
                      Sign in
                    </b>
                  </button>
                </div>
                <div
                  class="direct"
                >
                  <a
                    class=""
                    href="/register"
                  >
                    New here? Register now
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
